lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationServerLLGS.cpp

```diff
@@ -1436,6 +1452,21 @@ GDBRemoteCommunicationServerLLGS::Handle_jLLDBTraceGetBinaryData(
     return SendErrorResponse(bytes.takeError());
 }
 
+GDBRemoteCommunication::PacketResult
+GDBRemoteCommunicationServerLLGS::Handle_qStructuredDataPlugins(
+    StringExtractorGDBRemote &packet) {
+  // Fail if we don't have a current process.
+  if (!m_current_process ||
+      (m_current_process->GetID() == LLDB_INVALID_PROCESS_ID))
+    return SendErrorResponse(68);
```

Comment:
Does the `68` mean something special here? Can we use a named constant instead?

Reply:
I actually don't know what it means, but the entire file does that :(